.github/workflows/docker-build-push.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     name: Build services
     strategy:
       matrix:
-        service_name: [ "sample-service", "practitioner-service", "trial-config-service", "site-service", "role-service"]
+        service_name: [ "sample-service", "practitioner-service", "trial-config-service", "site-service", "role-service", "init-service"]
```

Comment:
why dont you remove the old 'trial config service'? I think this should be its replacement

Reply:
I would like for it to remain currently, we will raise a story to remove all traces of trial-config-service.

I see some config-server changes have been made to trials-config-service

https://github.com/NDPH-ARTS/mts-services/commit/1a68d22d2766b7ebe906fddcad01f8790ee134ca